6_temp_coop_fetch.yml

```diff
@@ -17,46 +17,36 @@ targets:
 
   6_temp_coop_fetch:
     depends:
-      - 6_temp_coop_fetch/downloads/long_term_ntl.csv.ind
-      # THIS IS FAILING TO DOWNLOAD - 6_temp_coop_fetch/downloads/mendota_daily_buoy.csv.ind
-      # THIS IS FAILING TO DOWNLOAD - - 6_temp_coop_fetch/downloads/mendota_temps_long.csv.ind
+
       - 6_temp_coop_fetch/log/6_temp_coop_fetch_tasks.ind
 
   # -- download LTER files and push to GD -- #
   coop_file_upload_location:
     command: as_id(I('1dutCiFEOoRObXLn6n3BRKhZDDAijK65Q'))
 
-  6_temp_coop_fetch/downloads/long_term_ntl.csv.ind:
-    command: download_and_indicate(in_ind = target_name,
-             url = I("https://lter.limnology.wisc.edu/sites/default/files/data/ntl29_v5.csv"),
-             push_location = coop_file_upload_location)
-
-  6_temp_coop_fetch/downloads/mendota_daily_buoy.csv.ind:
-    command: download_and_indicate(in_ind = target_name,
-              url = I("https://lter.limnology.wisc.edu/sites/default/files/data/ntl130_1_v7.csv"),
-              push_location = coop_file_upload_location)
-
-  6_temp_coop_fetch/downloads/mendota_temps_long.csv.ind:
-    command: download_and_indicate(in_ind = target_name,
-              url = I("https://lter.limnology.wisc.edu/sites/default/files/data/Lake_Mendota_temps_long.csv"),
-              push_location = coop_file_upload_location)
-
-
   # -- create makefile with targets for each file listed in gd -- #
-
-  # update dummy_date if you know there are updates in google drive (e.g., new cooperator file)
-  # that you want the peipline to recognize
+  # for these two targets, trigger_file is used to keep this _always stale_, so it always gets rebuilt.
+  # kind of unnecessary to have a cache file that is always stale - perhaps should only be local?
+  # but we're tacking the time onto the hash in a way that wouldn't make sense w/ a data file
   6_temp_coop_fetch/out/coop_all_files.rds.ind:
-    command: files_in_drive(out_ind = target_name, gd_path = coop_file_upload_location, dummy_date = I('2021-12-01'))
+    command: files_in_drive(out_ind = target_name,
+      gd_path = coop_file_upload_location,
+      trigger_file = '1_crosswalk_fetch/out/always_stale_time.txt',
+      trigger_wait = 30)
 
   coop_wants:
-    command: filter_coop_all('6_temp_coop_fetch/out/coop_all_files.rds.ind')
+    command: filter_coop_all('6_temp_coop_fetch/out/coop_all_files.rds.ind',
+      trigger_file = '1_crosswalk_fetch/out/always_stale_time.txt')
```

Comment:
As far as I can tell, this is related to the way remake is tracking build changes while things are executing. If we only have one target in the dependency chain that has the trigger file, it oddly is not considered stale the second build (even though you would expect it to, since it has secretly modified the trigger_file as a side-effect of the build). I can't remember exactly what I determined here - perhaps if you don't have two trigger_file-associated targets, you only get rebuilds every other time, which is frustrating and confusing. So my solution is to always pair them when used as this is reliable 100% of the time.